src/modules/organizations/containers/projects-container.jsx

```diff
@@ -16,6 +16,7 @@ class ProjectsContainer extends React.Component {
     super(props);
 
     this.state = {
+      meta: {},
```

Comment:
Can we change this to `meta: null` as default state? (empty object is throwing errors)